db/migrate/20230609134307_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+      t.text :test
```

Comment:
[REQUIRED] I believe you wanted to write `text` instead of `test`. Leaving it like this could cause confusion